env0/resource_project_policy.go

```diff
@@ -162,7 +162,7 @@ func resourcePolicyUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 		payload.DefaultTtl = ""
 	}
 
-	if payload.MaxTtl == "Infinine" {
+	if payload.MaxTtl == "Infinite" {
```

Comment:
This is what probably caused the issue. Because the acceptance was crashing it didn't catch the issue in the test.

Reply:
> Because the acceptance was crashing it didn't catch the issue in the test

@TomerHeber @yaronya Can we open an issue and follow up on fixing _that_, please?